lib/metadata_json_deps/runner.rb

```diff
@@ -25,12 +36,21 @@ def run
       end
       @updated_module = @updated_module.sub('-', '/')
       @module_names.each do |module_name|
-        message += "Checking *#{module_name}*\n"
+        message += "Checking *#{module_name}*.\n"
         module_name = module_name.sub('/', '-')
+
+        begin
+          @forge.get_mod(module_name)
+        rescue StandardError => error
+          message += "Checked module name *#{module_name.sub('-', '/')}* is INCORRECT. Please check managed_modules file! \n\n"
```

Comment:
Might be worth also changing this to something like "Checked module *#{module_name.sub('-', '/')}* could not be found. Verify *#{@updated_module}* exists on Puppet Forge"